Framework/Source/Utils/RenderGraphLiveEditor.cpp

```diff
@@ -1,5 +1,5 @@
 /***************************************************************************
-# Copyright (c) 2015, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
```

Comment:
This should be part of OS.h, there's nothing here that's actually editor code.

OS.h should provide:
- Function to create a memory-mapped file
- Function to register a callback for modified files

Then the RenderGraphViewer can just use those